service/router.go

```diff
@@ -46,6 +46,9 @@ func AdminRouter(env *Env) *mux.Router {
 	// Start the web service router
 	router := mux.NewRouter()
 
+	// API routes: csrf token
+	router.Handle("/v1/csrf", Middleware(http.HandlerFunc(TokenHandler), env)).Methods("GET")
```

Comment:
Can we call this /v1/token instead? I'm wondering if it should respond to PUT to make it a little more secure.